federation/pkg/federation-controller/util/meta_test.go

```diff
@@ -115,3 +115,23 @@ func TestObjectMetaAndSpec(t *testing.T) {
 	assert.False(t, ObjectMetaAndSpecEquivalent(&s1, &s3))
 	assert.False(t, ObjectMetaAndSpecEquivalent(&s2, &s3))
 }
+
+func TestIsFederationOnlyObject(t *testing.T) {
```

Comment:
(No action required) This test is verbose for what it does - things like name and namespace aren't required.  Also, it's generally preferable to define a test case for each call to the function under test rather than calling each one sequentially as is done here.